index.html

```diff
@@ -99,7 +99,7 @@ <h4>Workshop.</h4>
         <div class="program">
           <i class="far fa-handshake fa-2x"></i>
           <h4>E Ignite!</h4>
-          <p>Get opportunities to network with E-learning affliate around the world, also 
+          <p>Get opportunity to network with E-learning affliate around the world, also 
```

Comment:
* Kindly fix the footer section of the about page it needs a proper margin between the logo and text 
![image](https://user-images.githubusercontent.com/52497246/133491097-c4f6e401-3052-4a50-bf12-9555cedf1c13.png)